cli/src/cmd/forge/selectors.rs

```diff
@@ -89,6 +114,70 @@ impl SelectorsSubcommands {
+                    let target_path = canonicalize(&*contract_path)?;
+                    *contract_path = target_path.to_string_lossy().to_string();
+                    compile::compile_files(&first_project, vec![target_path], true)
+                } else {
+                    compile::suppress_compile(&first_project)
+                }?;
+
+                // Build second project
+                let second_project = build.project()?;
+                let second_outcome = if let Some(ref mut contract_path) = second_contract.path {
+                    let target_path = canonicalize(&*contract_path)?;
+                    *contract_path = target_path.to_string_lossy().to_string();
+                    compile::compile_files(&second_project, vec![target_path], true)
+                } else {
+                    compile::suppress_compile(&second_project)
+                }?;
+
+                // Find the artifacts
+                let first_found_artifact = first_outcome.find_contract(&first_contract);
+                let second_found_artifact = second_outcome.find_contract(&second_contract);
+
+                // Unwrap inner artifacts
+                let first_artifact = first_found_artifact.ok_or_else(|| {
+                    eyre::eyre!("Failed to extract first artifact bytecode as a string")
+                })?;
+                let second_artifact = second_found_artifact.ok_or_else(|| {
+                    eyre::eyre!("Failed to extract second artifact bytecode as a string")
+                })?;
+
+                // Check method selectors for collisions
+                let first_method_map = first_artifact.method_identifiers.as_ref().unwrap();
+                let second_method_map = second_artifact.method_identifiers.as_ref().unwrap();
+
+                let colliding_methods: Vec<(&String, &String, &String)> = first_method_map
+                    .iter()
+                    .filter_map(|(k1, v1)| {
+                        second_method_map
+                            .iter()
+                            .find_map(|(k2, v2)| if **v2 == *v1 { Some((k2, v2)) } else { None })
+                            .map(|(k2, v2)| (v2, k1, k2))
+                    })
+                    .collect();
+
+                if colliding_methods.is_empty() {
+                    println!("No colliding method selectors between the two contracts.");
+                } else {
+                    let mut table = Table::new();
+                    table.set_header(vec![
+                        String::from("Selector"),
+                        first_contract.name,
+                        second_contract.name,
+                    ]);
+                    colliding_methods.iter().for_each(|t| {
+                        table.add_row(vec![t.0, t.1, t.2]);
+                    });
+                    println!("{table}");
+
+                    return Err(eyre::eyre!("At least one collision was found"))
```

Comment:
Ok so what I'd do for this command is just not return an error at all—printing the table if we find collisions preceded by `x collisions found` is good enough for me. I'd only return an error if the command failed _internally_, as in building the project or trying to extract the selectors